### ISSUE TITLE
Updated Repository of World Names

### ISSUE THREAD

**Issue by TheWebfrog** [CleverRaven/Cataclysm-DDA#19292]
[THIS](https://en.wikipedia.org/wiki/List_of_New_England_towns) is a list of all the New England Towns, altough this is an ardous task it would be great to update the World name gen with more of the ACTUAL New England Towns, Plantations and Cities.

**Comment by Rod-Serling** [comment]
I took that list, copied it, stripped out all the extra text, and alphabetically sorted it.

Then I took the list from names/en.json, stripped the json formatting, and compared.

Names from names/en.json are on the left.
Names from the wiki are on the right.
https://www.diffchecker.com/FvtLBTsM

Looks like someone already used wiki to get the full list of names, as evidenced by Line 348 of the diff "Former town:" appearing in the list of names already in the game.  While this is an issue, the only other differences are in alphabet order, and three towns missing (Dana, Jeffersonville, and Wentworth's Location) two of which are former towns no longer incorporated (Dana and Wentworth's Location).

All in all the list of names is fine. I'll issue a PR shortly adding the three missing towns and removing the "Former town:" bug.
